resources/boundaries/osm/de.yaml

```diff
@@ -11,6 +11,118 @@
         "11": "suburb"
 
     overrides:
+        id:
+            relation:
+                "62782": "city" # Hamburg
+
+                # Cities with "admin_level" != 8
+                # and "de:place"=city
```

Comment:
shouldn't we read those tags ? (it can be done after, I'm completely ok for a temporary solution on this)

Reply:
I agree that this list seems too long to maintain. But in that specific case I couldn't find a proper documentation about `de:place`, so it's unclear whether it will be reliable in the future.

There are probably other tags that could be useful to type zones though (`place`, etc).
Should they be hard-coded in cosmogony ? Or maybe we could extend the `overrides` rules to define cases where such tag values should override `admin_level`

Reply:
hum yes it might be interesting to hard code the reading of the `place` tag (but with what precedence ?)
what do you think about this @nlehuby ?

anyway, we should discuss this in a separate issue, I'm fine to merge this PR with the hardcoded list.

Reply:
yes, we should definitely do something about the `place` tag but I don't know what for now ...
It requires even more local knowledge than the `boundary` tag
* place = city vs place = town vs place = municipality
* place = suburb vs place = neighbourhood vs place = quarter vs place = borough
* place = region vs place = state vs place = district vs place = province
* place vs de:place in Germany

Maybe we can also imagine some data analysis to check the volumetry between Cosmogony output and places uses in the countries where we know how this tags are used ?